src/test/kotlin/org/rust/ide/lineMarkers/CargoTestRunLineMarkerContributorTest.kt

```diff
@@ -201,6 +201,11 @@ class CargoTestRunLineMarkerContributorTest : RsLineMarkerProviderTestBase() {
         fn has_icon() { assert(true) } // - Test has_icon
     """)
 
+    fun `test custom test attribute`() = doTestByText("""
+        #[tokio::test(threaded_scheduler)]
```

Comment:
Let's add more tests to check different custom attributes like `tokio::test`, `custom_test`